npm-shrinkwrap.json

```diff
@@ -1,574 +0,0 @@
-{
```

Comment:
Isn't that what the API's `npm-shrinkwrap` is for?
There's also [this problem](https://github.com/Shyp/sails/pull/49#discussion_r65448264).
We've discussed this before – I don't think you'll find a shrinkwrap file in any reputable NPM module, and that's for a reason. It's a huge anti-pattern.